README.md

```diff
@@ -660,6 +660,33 @@ terraform state rm aws_elasticsearch_domain.index
 … to update the Terraform state so that it reflects the deletion of the
 Elasticsearch domain. Now running `make deploy` should succeed. 
 
+
+## `NoCredentialProviders` while running `make deploy`
+
+If you get
+
+```
+Failed to save state: failed to upload state: NoCredentialProviders: no valid providers in chain.
+…
+The error shown above has prevented Terraform from writing the updated state
+to the configured backend. To allow for recovery, the state has been written
+to the file "errored.tfstate" in the current working directory.
+
+Running "terraform apply" again at this point will create a forked state,
+making it harder to recover.
+```
+
+This happens when the temporary STS credentials expire while `terraform apply`
```

Comment:
The sentence starts starts with "If you get" so this paragraph needs to complete the sentence. The sentence doesn't need to start with "If you get" but there needs to be something before the literal block and the part before and after need to tie together.